### ISSUE TITLE
os.path.dirname(__file__) returns empty

### ISSUE THREAD

**Issue by Lexstok** [mollie/mollie-api-python#6]
In the examples there are 2 examples for writing to the so-called database (in this case, generate a text file). This did not work on windows (perhaps this has nothing to do with this being on windows, but more todo with looking into the current directory), I had to change the code from
    `os.path.dirname(__file__)`
to
    `os.path.dirname(os.path.abspath(__file__))`
to get it to work.

Link to stackoverflow question that helped me find this : http://stackoverflow.com/questions/7783308/os-path-dirname-file-returns-empty

**Comment by mollierobbert** [comment]
Thanks Alex, we've applied a belated fix.

**Comment by Lexstok** [comment]
Nice - I'm only a github beginner, so this helps to keep the motivation.

**Comment by gonzaabel** [comment]
I have the same problem and `os.path.dirname(os.path.abspath(__file__))` was the correct way to get this working on Windows (`os.path.dirname(__file__)`  works perfectly for Linux).
Thanks Lexstok!

**Comment by mvdpanne** [comment]
Hey Gonzalo,

Is this still a problem in the newest version of the API client? For example, in `examples/2-webhook-verification.py` and most of the other examples, the following is done:
```
sys.path.append(os.path.abspath(os.path.dirname(__file__) + '/../'))
```
Your comment seems to indicate it has to be changed to the following to make it work on Windows:
```
sys.path.append(os.path.dirname(os.path.abspath(__file__ + '/../')))
```

Both ways work on MacOS/Linux, so please let me know if this is a problem on Windows.

**Comment by gonzaabel** [comment]
Hi @mvdpanne , sorry but my problem wasn't of this repository hehehe.
But, answering your question, I can say after test it on Windows 10 x64 than `os.path.dirname(__file__)` returns empty path.
But, it works perfectly for me with Linux Mint & OpenSUSE. So when I have to test on Windows the project for I am working ([paxapos/fiscalberry](https://www.github.com/paxapos/fiscalberry)) it give me an error in another line. Because the path I wanted to use was empty.
Searching in Google, I find this issue and I try with `os.path.abspath(os.path.dirname(__file__))` like @Lexstok do... And after print the result, I can see the path 😃 .
So, in a nutshell, `os.path.abspath(os.path.dirname(__file__))` works for Windows, Linux, (and you have tested it on MacOS) as well.

Greetings :)

**Comment by mvdpanne** [comment]
Thanks for the clarification 👍
